shell/platform/windows/flutter_windows_unittests.cc

```diff
@@ -481,41 +546,28 @@ TEST_F(WindowsTest, Lifecycle) {
 }
 
 TEST_F(WindowsTest, GetKeyboardStateHeadless) {
-  // Run the test on its own thread so that it can pump its event loop while
-  // this thread waits.
-  fml::AutoResetWaitableEvent latch;
-  auto platform_task_runner = CreateNewThread("test_platform_thread");
-  platform_task_runner->PostTask([&]() {
-    auto& context = GetContext();
-    WindowsConfigBuilder builder(context);
-    builder.SetDartEntrypoint("sendGetKeyboardState");
-
-    bool done = false;
-    context.AddNativeFunction(
-        "SignalStringValue",
-        CREATE_NATIVE_ENTRY([&](Dart_NativeArguments args) {
-          auto handle = Dart_GetNativeArgument(args, 0);
-          ASSERT_FALSE(Dart_IsError(handle));
-          auto value = tonic::DartConverter<std::string>::FromDart(handle);
-          EXPECT_EQ(value, "Success");
-          done = true;
-          latch.Signal();
-        }));
-
-    ViewControllerPtr controller{builder.Run()};
-    ASSERT_NE(controller, nullptr);
+  auto& context = GetContext();
```

Comment:
The extra thread was unnecessary for this test. Consider reviewing this with whitespace changes disabled: https://github.com/flutter/engine/pull/51908/files?w=1#diff-f0b8369989e179cde8da71010c79f2c542f16c55f91e9d9aeae1a3fe769cf83dR548